docs/settingUp.md

```diff
@@ -23,6 +23,14 @@ The instructions in all parts of this document work for Linux, OS X, and Windows
    ```sh
    git remote add upstream https://github.com/TEAMMATES/teammates.git
    ```
+  
+ **Verification** : Use this command `git remote -v` and the output will be something like this :
```

Comment:
@wkurniawan07 So can i remove this or you want me to find an alternative for this ?